entity.ui.inc

```diff
@@ -461,7 +461,7 @@ class EntityContentUIController extends EntityDefaultUIController {
       'page arguments' => array($this->entityType, $this->id_count),
       'load arguments' => array($this->entityType),
       'access callback' => 'entity_access',
-      'access arguments' => array('edit', $this->entityType, $this->id_count),
+      'access arguments' => array('update', $this->entityType, $this->id_count),
```

Comment:
entity_access in Backdrop only allows 4 operations: 'create', 'view', 'update', 'delete'. Checking for 'edit' will always fail.